.meta.toml

```diff
@@ -0,0 +1,15 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
+[meta]
+template = "pure-python"
```

Comment:
Maybe the template `buildout-recipe` is better suited here as it provides infrastructure for coverage when using multiple processes.

Reply:
Nah, zope.testrunner is not a buildout recipe.  I'd rather add a flag to the pure-python recipe for supporting multiprocess coverage, I expect it'll come in handy for other packages (e.g. zdaemon).

Reply:
At least in the case of buildout recipes the coverage config has to be stored in a `.coveragerc` – that's the main difference to the `pure-python` template. Having both `.coveragerc` and coverage config in `tox.ini` in a template makes it even more complex.
Maybe the name of the `buildout-recipe` template is only too specific and it should be called `coverage-over-subprocesses` (?).